src/main/java/io/percy/selenium/Environment.java

```diff
@@ -1,4 +1,4 @@
-package io.percy.selenium;
```

Comment:
I think we should leave this package name as `io.percy.selenium`, because all the code in here is Selenium-specific, and not general Percy Java stuff.

My sense is that the `groupId` is sort of like the organization, and then each package within it is a different library (and the package tends to be aligned with the name of the `artifactId`. So I think we want a more granular package than `io.percy` for this library, unless we intend this to be the only Percy Java library, which would make this _the_ `io.percy` Java library (and then we would change the `artifactId` to be `percy-java`, minus the `selenium` part).

This way, if we ever build more Java SDKs, we'd have:
```
groupId=io.percy
|
|- artifactId=percy-java-selenium ; package io.percy.selenium
|
|- artifactId=percy-java-appium ; package io.percy.appium
|
...
```